lib/windows/winlean.nim

```diff
@@ -36,6 +36,8 @@ type
   DWORD* = int32
   PDWORD* = ptr DWORD
   LPINT* = ptr int32
+  ULONG_PTR* = uint
```

Comment:
And also CompletionKey is a value, not pointer, or you prefer do not use constants for CompletionKey?